homeassistant/components/switch/fritzdect.py

```diff
@@ -147,25 +193,45 @@ def __init__(self, fritz, ain):
         self.has_switch = STATE_UNKNOWN
         self.has_temperature = STATE_UNKNOWN
         self.has_powermeter = STATE_UNKNOWN
+        self.is_online = False
 
     def update(self):
         """Get the latest data from the fritz box."""
-        from requests.exceptions import RequestException
+        if self.is_online is False:
+            _LOGGER.error("Not online skipping request")
+            return
 
         try:
             actor = self.fritz.get_actor_by_ain(self.ain)
+        except (RequestException, HTTPError):
+            _LOGGER.error("Request to actor registry failed")
+            self.state = STATE_UNKNOWN
```

Comment:
Use `None` for unknown state. The base entity class will handle this.

Reply:
That is exactly what is happening: https://github.com/BastianPoe/home-assistant/blob/d1e730eb2fd376dfb76d9e845d20c301cd6715ec/homeassistant/components/switch/fritzdect.py#L91

Reply:
Replace `STATE_UNKNOWN` with `None.

Reply:
Done